### PR TITLE
Phase2-hgx315C Tests to investigate overlaps in the dd4hep version of HGCal

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Tests to investigate overlaps in the dd4hep version of HGCal

#### PR validation:

Use the scripts to compare DDD vs DD4hep usage for V17 HGCal geometry creation

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HGCalCommonData/plugins/DDHGCalWaferPartialRotated.cc [hunk 1]
```diff
@@ -22,7 +22,7 @@
 class DDHGCalWaferPartialRotated : public DDAlgorithm {
 public:
   // Constructor and Destructor
-  DDHGCalWaferPartialRotated();
+  DDHGCalWaferPartialRotated() {}
```
[the remark this answers is not in the document]
Comment (bsunanda) [reply Geometry/HGCalCommonData/plugins/DDHGCalWaferPartialRotated.cc]: Done

**Comment by srimanob** [comment]
+Upgrade

**Comment by civanch** [comment]
+1

@cvuosalo already signed

**Comment by qliphy** [comment]
+1

**Pull request merged**